apps/docusaurus/docs/guides/keyless-accounts.md

```diff
@@ -0,0 +1,199 @@
+    - in addition, an `email_verified` field will be exposed to indicate if the provider has verified that the user owns this email address
+  - `nonce`, arbitrary data that the application wants the OIDC provider to sign over
+  - `iat`, the time the the JWT was issued at.
+- **Ephemeral Key Pair:** a temporary public/private key pair that is used to sign transactions for an Aptos Keyless account. The public key and its expiration date are committed in the JWT token via the `nonce` field.
+- **Keyless Account:** a blockchain account that is directly-derived from (1) a user’s OIDC account (e.g., `alice@gmail.com`) and (2) an associated application’s OAuth client_id (e.g., Notion.so). Users authenticate through the OIDC flow.
+- **JSON Web Key (JWK):** is the cryptographic public key of the OIDC provider. This public key is used to verify the signature on the JWTs that the OIDC provider issues to the client application. This way, the client application can verify the authenticity of the tokens and ensure that they have not been tampered with.
+- **Pepper:** a secret value added to a key before hashing to preserve privacy.
+- **client_id:** the OAuth identifier for your application that you will receive from the IdP after registering your application with them. This will be used in our keyless architecture in the address derivation for your users.
+- **redirect_uri:** the URI of the callback handler once the user successfully authenticates. Needs to be registered with your IdP.
+
+# Keyless Account Integration Steps
+
+At a high level, there are three steps to follow in order to integrate Keyless Accounts.
+
+1. **Configure your OpenID integration with your IdP.** In this step, the dApp will register with the IdP of choice (e.g. Google) and receive a `client_id`
+2. **Install the Aptos TypeScript SDK.**
+3. **Integrate Keyless Account support in your application client**
+   1. Set up the `“Sign In with [Idp]”` flow for your user.
+   2. Instantiate the user’s `KeylessAccount`
+   3. Sign and submit transactions via the `KeylessAccount`.
+
+## Step 1. Configure your OpenID integration with your IdP
+
+### Supported Identity Providers
+
+Currently only Google is supported. We will support additional OIDC providers in the future (e.g., Apple, Kakaotalk, Microsoft, etc.).
+
+| Identity Provider | Auth URL                                                                                                                                                                       |
+| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Google            | https://accounts.google.com/o/oauth2/v2/auth/oauthchooseaccount?client_id=${CLIENT_ID}&redirect_uri=${REDIRECT_URI}&response_type=id_token&scope=openid%20email&nonce=${NONCE} |
+
+### Google
```

Comment:
My context may be old, but if not mention somewhere that dapps need to use the implicit OAuth flow